### PR TITLE
VC-1541 VC-1633 Update texts and add correct titles on controls

### PR DESCRIPTION
**Pull request by pavelthq**

### CODE REVIEW

**Review by konutis — APPROVED** [review]

File: visualcomposer/resources/views/settings/pages/system-status.php [hunk 1]
```diff
@@ -143,12 +143,12 @@
             </thead>
             <tbody>
             <tr>
-                <td><?php echo esc_html__('Plugin folder', 'visualcomposer') ?>:</td>
+                <td><?php echo esc_html__('Plugin directory', 'visualcomposer') ?>:</td>
```
Comment (FilipsMasolovs) [inline visualcomposer/resources/views/settings/pages/system-status.php:146]: Just a question, why is it directory here?

File: visualcomposer/resources/views/settings/pages/system-status.php [hunk 2]
```diff
@@ -143,12 +143,12 @@
...
                 <td class="vcv-help">
                     <span class="vcv-help-tooltip-icon"></span>
                     <span class="vcv-help-tooltip">
                         <?php echo esc_html__(
-                            'Plugin folder must be `visualcomposer` to ensure correct plugin work.',
+                            'Plugin folder must be `visualcomposer`.',
```
Comment (FilipsMasolovs) [inline visualcomposer/resources/views/settings/pages/system-status.php:151]: And folder here?

File: visualcomposer/resources/views/settings/pages/system-status.php (hunk 1 above)
Comment (pavelthq) [reply visualcomposer/resources/views/settings/pages/system-status.php:146]: @FilipsMasolovs https://visualcomposer-hq.slack.com/archives/C01DGMBK18A/p1607087277004200

File: visualcomposer/resources/views/settings/pages/system-status.php (hunk 2 above)
Comment (pavelthq) [reply visualcomposer/resources/views/settings/pages/system-status.php:151]: https://visualcomposer-hq.slack.com/archives/C01DGMBK18A/p1607087277004200

Comment (pavelthq) [reply visualcomposer/resources/views/settings/pages/system-status.php:151]: after discussion we decided to use folder everywhere

File: visualcomposer/resources/views/settings/pages/system-status.php (hunk 1 above)
Comment (pavelthq) [reply visualcomposer/resources/views/settings/pages/system-status.php]: after discussion we decided to use folder everywhere

**Review by FilipsMasolovs — APPROVED** [review]

**Pull request merged**